images/cassandra/config/main.tf

```diff
@@ -6,7 +6,7 @@ terraform {
 
 variable "extra_packages" {
   description = "The additional packages to install."
-  default     = ["cassandra", "cassandra-compat", "openjdk-11-default-jvm"]
+  default     = ["cassandra-4.1", "cassandra-4.1-compat", "openjdk-11-default-jvm"]
```

Comment:
Will this need to be updated when 4.2+ comes out? We can potentially automate this, but it'd be better to have this not require edits when the packages change.

Reply:
This is not needed as the packages provides latest virtuals